docs/src/scs/pki.md

```diff
@@ -25,6 +25,36 @@ HSM can be also used to perform cryptographic operations such as digital signing
 
 One example of affordable HSM solutions is YubiHSM, developed by Yubico.
 
+### HSM Variants for consideration
+
+The following HSM Solutions are considered for Ghaf project:
+
+  - [YubiHSM2](https://www.yubico.com/fi/product/yubihsm-2/ "YubiHSM2")
+  - [NitrokeyHSM2](https://shop.nitrokey.com/shop/product/nkhs2-nitrokey-hsm-2-7 "NitrokeyHSM2")
+  - [SoftHSMv2](https://github.com/opendnssec/SoftHSMv2 "SoftHSMv2")
```

Comment:
Do we need a link for BreadboardHSM as well?
It is mentioned in the table below.

Reply:
Not sure yet. We should think what to do with it.